react/features/virtual-background/components/VirtualBackgroundDialog.js

```diff
@@ -137,7 +139,7 @@ function VirtualBackground({ dispatch, t }: Props) {
             hideCancelButton = { true }
             submitDisabled = { false }
             titleKey = { 'virtualBackground.title' }
-            width = 'small'>
+            width = '450px'>
```

Comment:
How many icons fit in a single row now?

Reply:
8. Changed the upload limit to 24 (3 rows of uploaded images) Also I found a bug. I fixed it with a try/catch statement to prevent the localStorage QUOTA_EXCEEDED_ERR.